backend/dataall/modules/s3_datasets_shares/aws/s3_client.py

```diff
@@ -49,6 +49,9 @@ def create_bucket_access_point(self, bucket_name: str, access_point_name: str):
                 Bucket=bucket_name,
             )
         except Exception as e:
+            if 'AccessPointAlreadyOwnedByYou' in str(e):
+                log.info(f'S3 bucket access point {access_point_name} already exists on {self._account_id}')
+                return self.get_bucket_access_point_arn(access_point_name)
             log.error(f'S3 bucket access point creation failed for location {bucket_name} : {e}')
```

Comment:
nit: would convert to `log.exception(S3 bucket access point creation failed for location {bucket_name})`